kernel/realsense/d4xx.c

```diff
@@ -4997,6 +5000,7 @@ static int ds5_dfu_device_open(struct inode *inode, struct file *file)
 	i2c_set_adapter_bus_clk_rate(parent, DFU_I2C_BUS_CLK_RATE);
 #endif
 #endif
+	mutex_unlock(&state->lock);
```

Comment:
Who reads `state->dfu_dev.bus_clk_rate`?

We set using the i2c API right?
Why do we need to keep the last value? can't we hard coded set 400 on entrance and set 100 on exit?

Reply:
we can hard code values